src/utils/report.ts

```diff
@@ -28,10 +28,24 @@ function recordConver (args: ConverObj) {
   })
 }
 
+function _sortByNum (reportList: ConverObj[] = []) {
+  reportList.sort((current, next) => {
+    // same char, then sort by after-number
+    if (current.num[0] === next.num[0]) {
+      return +current.num.substring(1) - +next.num.substring(1)
```

Comment:
We can make this more readable
```suggestion
      return Number(current.num.substring(1)) - Number(next.num.substring(1))
```